### PR TITLE
Add master, nightly and release scripts for centOS CI jobs

### PR DESCRIPTION
**Pull request by sleshchenko**
### What does this PR do?
This PR removes scripts that were used by CodenvyCI and adds master, nightly and release scripts for centOS CI jobs.
The corresponding jobs are added in https://github.com/openshiftio/openshiftio-cico-jobs/pull/1105
### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/14378

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+3.1.0-SNAPHOT
```
Comment (ibuziuk) [inline VERSION]: has `3.0.0` version been released already?

Comment (sleshchenko) [reply VERSION]: Not yet. I'm going to do it as soon as webhooks will be set.
Do you think it makes sense to leave 3.0.0-SNAPSHOT now?

Comment (ibuziuk) [reply VERSION]: yeah, IMO ` 3.0.0-SNAPSHOT` is a better choice

**Comment by sleshchenko** [comment]
I'm merging this PR since Draft with jobs is already merged. But feel free to comment and I'll address it in upcoming  PRs.

**Pull request merged**